src/App.tsx

```diff
@@ -1,61 +1,121 @@
+import { useState } from 'react';
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo, User, TodosWithUser } from './types/Types';
```

Comment:
Add types to the PR
![image](https://user-images.githubusercontent.com/91411692/237056089-2acffba3-39ee-4d51-be06-eda139eac747.png)